### PR TITLE
feat(monitors): Add analytics on page view

### PR DESCRIPTION
**Pull request by davidenwang**

### CODE REVIEW

File: static/app/views/monitors/monitors.tsx [hunk 1]
```diff
@@ -37,6 +38,10 @@ type State = AsyncView['state'] & {
 class Monitors extends AsyncView<Props, State> {
   getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
     const {params, location} = this.props;
+
+    trackAdvancedAnalyticsEvent('monitors.page_viewed', {
+      organization: params.orgId,
+    });
```
Comment (billyvg) [inline static/app/views/monitors/monitors.tsx:44]: Hmm I don't think it's great to add a side-effect to `getEndpoints` -- it'd be better to stick this in a lifecycle method

**Comment by scttcper** [comment]
cc @scefali

File: static/app/views/monitors/monitors.tsx (hunk 1 above)
Comment (scefali) [reply static/app/views/monitors/monitors.tsx:44]: Also this is going to break because `params.orgId` is a slug and not an actual ID, right?

Comment (scefali) [reply static/app/views/monitors/monitors.tsx:44]: we have the full `organization`, let's use that

Comment (davidenwang) [reply static/app/views/monitors/monitors.tsx:44]: ahh didn't know about the slug/id, but will change accordingly

**Review by billyvg — APPROVED** [review]

**Pull request merged**